### DISCUSSION TITLE
Copy folder with content

### DISCUSSION THREAD

**Discussion by stefanstranger** [PowerShellOrg/Plaster#407] [Ideas]
Instead of a single file I want to copy complete folders with content. Is that possible too?

**Comment by davegreen** [comment]
I don't think you can copy a folder and it's files, although I know you can copy groups of files to a target folder, some of the documentation about [creating a manifest](https://github.com/PowerShell/Plaster/blob/master/docs/en-US/about_Plaster_CreatingAManifest.help.md#content-element-file) lists a relevant (but not quite right) example.

There's a possibility it's only a small change to add this functionality, or using wildcards will do the trick.

**Comment by terrabitz** [comment]
Try using a double asterisk:

```none
<file source='myfolder\**' destination="myfolder" />
```

I have had some success with this, even though it's not explicitly mentioned anywhere in the docs above. If this is intended behaviour, it should probably be added.
